src/util/namespacesUtils.ts

```diff
@@ -0,0 +1,123 @@
+// import { readonly, computed } from "vue";
+import {
+  NamespaceMosaicIdGenerator,
+  NetworkType,
+  RegisterNamespaceTransaction,
+  UInt64,
+} from "tsjs-xpx-chain-sdk";
+// import { mergeMap, timeout, filter, map, first, skip } from 'rxjs/operators';
+import { walletState } from "../state/walletState";
+import { networkState } from "../state/networkState";
+import { ChainUtils } from "../util/chainUtils";
+import { WalletUtils } from "../util/walletUtils";
+import { ChainAPICall } from "../models/REST/chainAPICall";
+import { BuildTransactions } from "../util/buildTransactions";
+import { Helper } from "./typeHelper";
+import { WalletAccount } from "@/models/walletAccount";
+
+
+export class NamespacesUtils {
+
+  static getRootNamespaceTransactionFee =(networkType: NetworkType, generationHash: string, namespaceName: string, duration: number) :number => {
+    let buildTransactions = new BuildTransactions(networkType, generationHash);
+    const registerRootNamespaceTransaction = buildTransactions.registerRootNamespace(namespaceName, UInt64.fromUint(duration));
+    return registerRootNamespaceTransaction.maxFee.compact();
+  }
+
+  static getSubNamespaceTransactionFee =(networkType: NetworkType, generationHash: string, namespaceName: string, subNamespace: string) :number => {
+    let buildTransactions = new BuildTransactions(networkType, generationHash);
+    const registerSubNamespaceTransaction = buildTransactions.registersubNamespace(namespaceName, subNamespace);
+    return registerSubNamespaceTransaction.maxFee.compact();
+  }
+
+  static calculateDuration = (duration: number): number =>{
```

Comment:
maybe the param name be more specific ? durationInDay

Reply:
Sure, noted